app/templates/publications/publication/case.hbs

```diff
@@ -16,6 +16,7 @@
     />
 
     <Utils::GovernmentAreas::GovernmentAreasPanel
+      @datetimeForGovernmentAreas={{or this.agenda.created this.meeting.plannedStart this.model.created}}
```

Comment:
Geplande start van de meeting (indien er een is) lijkt mij correcter, en dus belangrijker, dan `agenda.created`. Dat laatste is een puur technische datum wanneer de gebruiker de agenda heeft aangemaakt in het systeem.